spyderlib/config/main.py

```diff
@@ -602,6 +602,8 @@ def is_gtk_desktop():
              {
               'names': ['Emacs', 'IDLE', 'Monokai', 'Pydev', 'Scintilla',
                         'Spyder', 'Spyder/Dark', 'Zenburn'],
+              'custom_names': [],

```

Comment:
I think we don't need this option here. Remember that we only add options to `config/main` if we plan to change and/or update them in the future :-)